C-Plus-Plus/ds/linked_list_middle_element.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+
+ Created by Sarthak-9
+ A Linked List is a linear data structure which includes a series of connected nodes. Each node stores the data and the address of the next node.
+
+ Middle Element can be extracted using following two methods :
+ Time complexity of MiddleNode1 : O(n)
+ Time complexity of MiddleNode2 : O(n/2)
+ Space Complexity of both : O(1)
+
+*/
+
+#include <iostream>
+#include <cmath>
+#include <stack>
+
+using namespace std;
+
+class Node{
+        public:
+                int data;
+                Node* next;
+};
```

Comment:
What level of indentation are you following? This is incoherent with the code that exists in the repo.